ansible/roles/ocp4-workload-logging/defaults/main.yml

```diff
@@ -3,8 +3,14 @@ become_override: False
 ocp_username: opentlc-mgr
 silent: False
 
-_logging_elasticsearch_replicas: 3
-_logging_elasticsearch_memory_request: "4Gi"
+_logging_elasticsearch_replicas: 1
+_logging_elasticsearch_memory_request: "8Gi"
 _logging_elasticsearch_storage_request: "200Gi"
```

Comment:
Can and should we make this smaller...and get curator configured to purge after like 3 or 5 days? Assuming curator works.

Reply:
Curator is configured quite conservatively. Like 2 days. There's a curator config map